<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWagonForm</class>
 <widget class="QMainWindow" name="AddWagonForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Менеджер поездов</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(50, 50, 50);
color: #b1b1b1;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="wagonName">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер вагона:</string>
    </property>
   </widget>
   <widget class="QLabel" name="trainName">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(75, 75, 75);
	border-radius: 10px;
}
QPushButton:hover {
     background-color: rgb(80,80, 80);
}
QPushButton:pressed {
    background-color: rgb(120,120,120);
}
</string>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(75, 75, 75);
	border-radius: 10px;
}
QPushButton:hover {
     background-color: rgb(80,80, 80);
}
QPushButton:pressed {
    background-color: rgb(120,120,120);
}
</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер поезда:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
